.eslintrc.cjs

```diff
@@ -380,12 +376,6 @@ module.exports = {
         ],
       },
     },
-    {
-      files: ["site/**"],
-      rules: {
-        "unicorn/filename-case": "error",
-      },
-    },
```

Comment:
I forgot to remove this in https://github.com/blockprotocol/blockprotocol/pull/305